### PR TITLE
Added power up feature. Players speed up and flicker after defending

### PR DESCRIPTION
**Pull request by senadkurtisi**
Flickering is just an indicator for the player/s that their power-up is still active.

### CODE REVIEW

File: source/dagger/gameplay/ping_pong/power_up.h [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#pragma once
+#include "core/system.h"
+#include "core/core.h"
+
+using namespace dagger;
+
+struct PowerUp
```
Comment (DPrivalov) [inline source/dagger/gameplay/ping_pong/power_up.h]: place it inside ping_pong namespace as any other pingpong system

**Review by DPrivalov — CHANGES_REQUESTED** [review]
Good job! 

One important thing to change and it's good to merge!

**Review by DPrivalov — APPROVED** [review]

**Review by UrosBorovcanin — COMMENTED** [review]
The code functions as it should and functions well. The only negative side that i see here is that it doesn't conform to the styleguide.md

**Review by DPrivalov — APPROVED** [review]

**Review by UrosBorovcanin — APPROVED** [review]
The code is up to the style guide standard and still works completely fine.

**Comment by DPrivalov** [comment]
@Djafex, waiting for u =)

**Review by FedjaVujicic — APPROVED** [review]

**Pull request merged**